packages/calcite-components/src/components/meter/meter.e2e.ts

```diff
@@ -0,0 +1,19 @@
+import { accessible, renders, hidden, t9n } from "../../tests/commonTests";
+
+describe("calcite-meter", () => {
+  describe("renders", () => {
```

Comment:
There might be other test helpers that could be used here (e.g., `defaults`, `reflects`).

Reply:
Added!